store/db/mysql/migration/prod/0.17/02__extend_text.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE `system_setting` MODIFY `value` LONGTEXT NOT NULL;
```

Comment:
Query 1 ERROR at Line 1: : near "MODIFY": syntax error

<img width="711" alt="image" src="https://github.com/usememos/memos/assets/24653555/9289c871-eb87-48ca-ac29-e3c18f5eb91e">

Reply:
It seems there isn't any syntax error in this SQL statement, what's going on?